dbt_subprojects/dex/models/trades/optimism/platforms/velodrome_optimism_base_trades.sql

```diff
@@ -35,6 +35,19 @@ dexs_v2 AS (
             pair_column_name = 'pool'
         )
     }}
+),
+
+dexs_v2_cl AS (
+        {{
+        uniswap_compatible_v3_trades(
+            blockchain = 'optimism',
+            project = 'velodrome',
+            version = '2',
```

Comment:
You may use another version value here to identify the source.

Reply:
Will review it and get it fixed. Thanks for the readme link! Sorry for missing it

Reply:
Maybe `cl`?
```suggestion
            version = 'cl',
```

Reply:
maybe v2_cl since its under v2

Would this break data types? iirc version contains integers?

Reply:
For consistency, maybe `2_cl`:
```suggestion
            version = '2_cl',
```